Bottom: 6dcde7d30e37f5c022655c7c37ae29b95e7fb1a2
Top:    be4804d68af5d78b9644864d04ef4d4797cf1663
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-25 01:14:32 +0100

Refresh of hid-gtk-implement-an-api-to-dr

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index ea1336e..2a7c145 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1325,7 +1325,7 @@ draw_lead_user (render_priv *priv)
   glColor3f (LEAD_USER_COLOR_R, LEAD_USER_COLOR_G,LEAD_USER_COLOR_B);
 
 
-  /* arcs at the approrpriate radiuses */
+  /* arcs at the approrpriate radii */
 
   for (i = 0; i < LEAD_USER_ARC_COUNT; i++, radius -= separation)
     {
